server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
 const http = require('http');
 
-// Creating the HTTP server
 const server = http.createServer((req, res) => {
-
```

Comment:
The MongoDB URI is hardcoded. It would be better to use environment variables to store sensitive information and retrieve them here.